### DISCUSSION TITLE
This address does not have a supported Ethereum name that resolves to it?

### DISCUSSION THREAD

**Discussion by simonecamilla** [smartcontractkit/full-blockchain-solidity-course-py#240] [Q&A]
When i transfered the ether to my rinkeby test network it is confirmed on https://faucet.rinkeby.io/ but it did not appear in my wallet. Then if i look on etherscan on my wallet address for the rinkeby test network wallet it says following:

"This address does not have a supported Ethereum name that resolves to it."

Anyone understands this and what I should do?

Hope everything is going good for everyone :)

**Comment by cromewar** [comment]
I had the same problem before so I recommend to use another faucet like chainlink: https://faucets.chain.link/rinkeby

**Reply by simonecamilla** [reply]
Thanks tried! unfortunately I did it multiple times and still nothing on my account even though confirmation.  hmm... do you know why?

**Reply by cromewar** [reply]
Could you post a screenshot of the chainlink transaction here please, it could give us more info about how the transaction went.

**Reply by simonecamilla** [reply]
![RequestComplete](https://user-images.githubusercontent.com/45121577/138362483-52f2ad79-3e78-49c0-9e78-fe455cf1f135.PNG)
![Transaction](https://user-images.githubusercontent.com/45121577/138362496-94df947d-939c-4a72-9c10-f21a3dff1e12.PNG)

**Reply by simonecamilla** [reply]
made a new one now again, didnt work!
i am completely new to this, thanks a lot for helping

**Reply by cromewar** [reply]
Tha's weird I cannot find anything strange on your transaction, are you sure you are using the correct wallet address? have you tried with another one?

**Reply by simonecamilla** [reply]
yess i tried with other test networks and i also tried creating a second account. Still doesn't work... i dubble checked a lot of times now everything from codes and all the steps... hmm

**Reply by cromewar** [reply]
Well funding a wallet has nothing to be with the code itself (at least when asking a faucet) so this is just an hypothesis but, could you try to install metamask on a new browser (brave or chrome) and create another wallet, then try again?

**Reply by simonecamilla** [reply]
So i tried it on brave and chrome! but here i found smt maybe this could be what is the problem
![not connected](https://user-images.githubusercontent.com/45121577/138516686-97ece43b-af16-4f58-9092-68973ca1837f.PNG)
?

**Reply by simonecamilla** [reply]
if i press the top there not connected this comes up,  i don't know how to connect it, didn't really see it before since i didn't use extension haha just looked on the webpage

**Reply by cromewar** [reply]
You don't need to connect to the site in order to get eth, I'm really lost on this one, I'm going to do deep research about your case, hope I'll find something.

**Reply by simonecamilla** [reply]
thank you so much!

**Reply by PatrickAlphaC** [reply]
@cromewar is doing amazing :)

**Reply by cromewar** [reply]
Well @simonecamilla  I did more research on your case but unfortunately i didn't find anything valuable, it seems this is a very case specific situation I have a last request for you. 

Please, install metamask wallet of your cellphone and create a new account there, try to sen some eth with chainlink faucet (be sure you change to the correct network) and please post here the result.

**Reply by cromewar** [reply]
Also you could try with this faucet: https://rinkeby-faucet.com/

Finally if you want i could send you some Eth to your Rinkeby address if you share it here.
